js/script.js

```diff
@@ -0,0 +1,20 @@
+let mobileMenu = document.getElementById('hamburger-div');
+let mobileMenuCancel = document.getElementById('cancel-hamburger');
+let mobileMenuCancel1 = document.getElementById('cancel-butt1');
+let mobileMenuCancel2 = document.getElementById('cancel-butt2');
+let mobileMenuCancel3 = document.getElementById('cancel-butt3');
```

Comment:
Using `querySelector` is more modern than `getElementById`. We should right modernize code as much as we can